src/Compatibility/Microsoft.DotNet.ApiCompatibility/Logging/SuppressionEngine.cs

```diff
@@ -67,11 +69,11 @@ public bool IsErrorSuppressed(Suppression error)
                 {
                     return true;
                 }
-                else if (error.DiagnosticId == null || error.DiagnosticId.StartsWith("cp", StringComparison.InvariantCultureIgnoreCase))
+                else if (error.DiagnosticId.StartsWith("cp", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
see my response above.

Reply:
I've replied above as well, I think this should be reverted since, independently of the fact that this worked or not, having a global suppression for a tfm pair is desired.